docs/api/index.md

```diff
@@ -312,6 +312,23 @@ When you use `test` in the top level of file, they are collected as part of the
   describe.todo.concurrent(/* ... */) // or describe.concurrent.todo(/* ... */)
   ```
 
+### describe.random
+
+- **Type:** `(name: string, fn: TestFunction, timeout?: number) => void`
+
+  Vitest provides a way to run all tests in random order via CLI flag [`--random`](/guide/cli) or config option [`sequence.random`](/config/#sequence-random), but if you want to have only part of your test suite to run tests in random order, you can mark it with this flag.
+
+  ```ts
+  describe.random('suite', () => {
```

Comment:
If so, should we also support `repeat` for `describe`? For example:

describe.repeat(10).random('repect 10 times randomly', () => {
  // ...
})

Reply:
How is this has anything to do with random? 👀

Reply:
No necessary in this PR. Using random feels more unstable to identify the ordering bug (like sometimes the change passes the CI because the order does not hit the bug, but will later). Running random tests multiple times could reduce such scenarios and make the tests more stable.

Reply:
> No necessary in this PR, just an idea. Using random feels more unstable to identify the ordering bug (like sometimes the change passes the CI because the order does not hit the bug, but will later). Running random tests multiple times could reduce such scenarios and make the tests more stable.

Sure, but this also should be done on the global level, yes? This PR adds `vitest --random`, so there should also be `vitest --random --repeat 10`.

It's also not clear what should be done on error. Do we fail the suite when it hits the first error? Do we show summary of all 10 runs? I think if we want to add this, it's better be done in other PR, and if/when people will ask for it.

I really want to switch to `@vitest/browser`, but all this enhancements PRs stop me 😄

Reply:
Sure we could do that later 👍